style.css

```diff
@@ -504,7 +744,371 @@ ul {
+    width: 300px;
+    height: 80px;
+  }
+
+  .description {
+    margin-top: -0.25rem;
+    width: 85%;
+    margin-left: -5.3rem;
+  }
+
+  .languages-list {
+    margin-left: -5.3rem;
+    margin-top: -8.5rem;
+  }
+
+  .buttons {
+    margin-left: -5.4rem;
+  }
+
+  .about-me {
+    display: grid;
+    flex-direction: row;
+    width: 100%;
+    height: 90vh;
+    padding: 56px 14px 0 16px;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-left: -2rem;
+  }
+
+  .about-text {
+    display: flex;
+    white-space: nowrap;
+    margin-left: 10%;
+    width: 100%;
+    margin-top: -2.9rem;
+  }
+
+  .summary {
+    margin-left: 10%;
+    width: 27.2rem;
+    height: 102px;
+    margin-top: -2rem;
+    font-family: "Poppins", sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+  }
+
+  .connection2 {
+    margin-left: 10%;
+    margin-top: 0.5rem;
+  }
+
+  .social2 {
+    margin-left: 7%;
+  }
+
+  .buttons2 {
+    margin-left: 10%;
+  }
+
+  .skill-list {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    float: right;
+    margin-left: 35rem;
+    margin-top: -45rem;
+  }
+
+  .down-arrow {
+    margin-left: 40rem;
+  }
+
+  .languages-container {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages-list2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-bottom: 3rem;
+  }
+
+  .javascript2 {
+    display: flex;
+    flex-direction: column;
+    width: 7.5rem;
+    height: 8rem;
+  }
+
+  .js-logo {
+    height: 4.5rem;
+    width: 4.4rem;
+  }
+
+  .js-text {
+    align-self: center;
+    margin-top: 5.6rem;
+    margin-left: -1.5rem;
+  }
+
+  .html2 {
+    display: flex;
+    flex-direction: column;
+    margin-top: -0.1rem;
+    width: 7.5rem;
+    height: 8rem;
+  }
+
+  .html-logo {
+    height: 4.5rem;
+    width: 4.4rem;
+  }
+
+  .html-text {
+    align-self: center;
+    margin-top: 5.8rem;
+    margin-left: -2.69rem;
+  }
+
+  .css2 {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0.021rem;
+    width: 7.5rem;
+    height: 8rem;
+  }
+
+  .css-logo {
+    height: 4.5rem;
+    width: 4.4rem;
+  }
+
+  .css-text {
+    align-self: center;
+    margin-top: 5.5rem;
+    margin-left: -3.4rem;
+  }
+
+  .frameworks-text {
+    margin-left: 46.2rem;
+    margin-top: 6.2rem;
+  }
+
+  .right-arrow {
+    margin-left: -2rem;
+    margin-top: 6.4rem;
+  }
+
+  .right-arrow2 {
+    margin-left: -5.2rem;
+    margin-top: 11rem;
+  }
+
+  .separator-bottom {
+    margin-left: -32rem;
+    margin-top: -1px;
+  }
+
+  .skills-text {
+    margin-left: 46.2rem;
+    margin-top: 9.7rem;
+  }
+
+  .separarot-bottom2 {
+    margin-left: -35.5rem;
+    margin-top: -1rem;
+  }
+
+  .contact-form {
+    width: 100%;
+    background-size: cover;
+    background-image: url(image/contact-form-background.svg);
+  }
```

Comment:
- [ ] Please consider aligning the content of the contact section correctly as shown in the Figma Design 👍🏼 
- [ ] The submit button should be exactly centered according to the form 👍🏼 

| Your Design | Figma Design |
|---|---|
|![image](https://user-images.githubusercontent.com/68030297/230695347-519a53f1-cd36-4501-b9c8-98c24d246e03.png) | ![Screenshot from 2023-04-08 01-29-11](https://user-images.githubusercontent.com/68030297/230695382-afe26426-19af-46c9-9acf-ac2e9c9323c6.png)|